### DISCUSSION TITLE
links/

### DISCUSSION THREAD

**Discussion by giscus** [By-Seayay/Blog#4] [Giscus]
# links/

友链申请欢迎交换友链，请按照以下格式提供您的站点信息，通过 Telegram 或 Giscus 评论区发送给我，我将尽快给予您答复。 1234567- url: # 站点链接  avatar: # 头像链接  name: # 站长名称  blog: # 站点名称  desc: # 站点介绍  color: '#66CCFF' # 代表色, 选填, 不填默认取头像色  email

https://blog.1kyr.top/links/

**Comment by Seayay** [comment]
本站的友链信息如下：
```
- url: "blog.1kyr.top"
  avatar: "https://avatars.githubusercontent.com/u/92374733?s=400&u=9acc95d2b39d9a08e9edbef553e9dcac29e801ba&v=4"
  name: "思遥方"
  blog: "思遥方的小家"
  desc: "翻译 · 创作 · 日常"
  color: "#ECAFD3"
  email: "siyaofang@126.com"
```

**Comment by HeimNad** [comment]
- url: "northlank.top"
  avatar: "https://northlank.top/logo.ico"
  name: "IMMG"
  blog: "北极熊地图管理组网站"
  desc: 查询, 提取, 高速下载北极熊翻译地图"
  color: ""
  email: ""

**Reply by Seayay** [reply]
已添加
